file-system-view.js

```diff
@@ -17,6 +17,7 @@ var View = function(id) {
 	this.path = cfg.path;
 	this.sep = require("path").sep;
 	this.history = [this.path];
+	this.showHiddenFiles = true;
```

Comment:
To init toggle state use `applicationController.config.settings.showHiddenFiles`